.github/workflows/update-github-pages.yaml

```diff
@@ -0,0 +1,27 @@
+name: Deploy github pages
+on:
+    workflow_dispatch:
+    push:
+        branches:
+            - master
```

Comment:
Either remove this or remove the step from deploy-prod, we don't need to do it twice!